ibm_catalog.json

```diff
@@ -19,35 +19,35 @@
           "title": "Creates a Secrets Manager instance.",
-          "description": "Creates an IBM Secrets Manager instance."
+          "description": "For more details on an IBM Cloud Secrets Manager instance, [see here](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-getting-started)."
         },
         {
           "title": "Create secret groups.",
-          "description": "Optionally create secret groups inside your IBM Secrets Manager instance."
+          "description": "For more details on creating optional secret groups inside the Secrets Manager instance, [see here](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-secret-groups&interface=ui)."
         },
         {
           "title": "Create access groups for your secret groups.",
-          "description": "Optionally create access groups for the secret groups inside your IBM Secrets Manager instance."
+          "description": "For more details on optional configuring access groups for the secret groups, [see here](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-assign-access#assign-access-secret-group-console)."
         },
         {
-          "title": "Optionally configure an IBM Secrets Manager IAM credentials engine to an IBM Secrets Manager instance.",
-          "description": "Optionally configure an IBM Secrets Manager IAM credentials engine to an IBM Secrets Manager instance."
+          "title": "Configure an IAM credentials engine to an IBM Secrets Manager instance.",
+          "description": "For more details on optionally configure an IBM Secrets Manager IAM credentials engine to an IBM Secrets Manager instance, [see here](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-configure-iam-engine&interface=ui)."
         },
         {
           "title": "Sets up authorization policy.",
           "description": "Sets up IBM IAM authorization policy between IBM Secrets Manager instance and IBM Key Management Service (KMS) instance. It also supports Event Notification authorization policy."
         },
         {
-          "title": "Configures lifecycle notifications for the Secrets Manager instance.",
-          "description": "Configures lifecycle notifications for the IBM Secrets Manager instance by connecting an IBM Event Notifications service. The automation supports optionally creating a KMS key ring and key, or using an already existing one to encrypt data."
+          "title": "Configures lifecycle notifications for the Secrets Manager instance using an IBM Event Notifications service.",
+          "description": "For more details on optionally configuring lifecycle notifications for the IBM Secrets Manager instance by connecting an IBM Event Notifications service, [see here](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-event-notifications&interface=ui). The automation supports optionally creates a KMS key ring and key, or using an already existing one to encrypt data."
```

Comment:
```suggestion
          "description": "Optionally, you can choose to configure lifecycle notifications by integrating the Event Notifications service. [Learn more](https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-event-notifications&interface=ui)."
```

Reply:
The second line after the link seems like an integration with KP that should be talked about separately. Maybe replace the getting started section with KP??